src/app/ui/bridge/utils.tsx

```diff
@@ -17,7 +17,13 @@ export function buildExplorerLink(
       color="text.primary"
       component={Link}
       target="_blank"
-      href={`${blockExplorerUrls[0]}/tokens/${address}`}
+      href={`${blockExplorerUrls[0]}tokens/${address}`} // todo: on OP it needs to be "token/..."
```

Comment:
It seems that the `/` prefix is not there anymore

Reply:
Well spotted! But I did this change on purpose -->

This was meant to be a fix for the linking to prevent 404s. Please check the linking yourself for the Token-address within the "Token Information" boxes on a page like this: https://jumper.exchange/bridge/optimism-usdt-to-zksync-eth

Some more links on other chains:
https://explorer.cronos.org**//**tokens/0x66e428c3f67a68878562e79A0234c1F83c208770
https://etherscan.io**//**tokens/0xdAC17F958D2ee523a2206206994597C13D831ec7
https://bscscan.com**//**tokens/0x55d398326f99059fF775485246999027B3197955
https://bobascan.com**//**tokens/0x5DE1677344D3Cb0D7D465c10b72A8f60699C062d

![image](https://github.com/user-attachments/assets/794cd6f8-1722-4873-84aa-e1a19fb0c220)